File: ABC/003/B.py

def check(S, T):
  possible = ['a', 't', 'c', 'o', 'd', 'e', 'r']
  for i in range(len(S)):
    if S[i] == T[i]: continue
    elif S[i] == '@' and T[i] in possible: continue
    elif T[i] == '@' and S[i] in possible: continue
    return False
  return True

S = raw_input()
T = raw_input()

if check(S, T): print 'You can win'
else: print 'You will lose'

File: ABC/003/B.rb
s = gets.chomp.to_s
t = gets.chomp.to_s
u = ['a', 't', 'c', 'o', 'd', 'e', 'r']
check = true
for i in 0...s.size
  next if s[i]==t[i]
  if s[i]=='@' && u.include?(t[i])
    next
  elsif t[i]=='@' && u.include?(s[i])
    next
  end
  check=false
  break
end

if check
  puts 'You can win'
else
  puts 'You will lose'
end
